arm9/src/scrcmd_amity_square.c

```diff
@@ -0,0 +1,117 @@
+#include "scrcmd.h"
+#include "constants/pokemon.h"
+#include "event_data.h"
+
+extern void FUN_0205F54C(struct ScriptState* state);
+extern u16 FUN_0205F55C(struct ScriptState* state);
+
+static const u16 sAmitySquarePossibleAccessoriesByPokemon[6][10] = {
+    // TODO: accessory constants
```

Comment:
if these constants are known, it would be good to get them done now before they're merged